boards/shields/nrf2220ek/boards/nrf52840dk_nrf52840.conf

```diff
@@ -0,0 +1,3 @@
+CONFIG_BOOT_BANNER=n
+CONFIG_EARLY_CONSOLE=n
+CONFIG_NCS_BOOT_BANNER=n
```

Comment:
don't see what this has to do with a shield?

Reply:
see https://github.com/nrfconnect/sdk-nrf/pull/19042#discussion_r1856226312